web/src/containers/activity/CostAllocateFFP.test.js

```diff
@@ -1,34 +1,41 @@
 import { shallow } from 'enzyme';
-import sinon from 'sinon';
 import React from 'react';
 
 import {
   CostAllocateFFPRaw as CostAllocateFFP,
   makeMapStateToProps,
   mapDispatchToProps
 } from './CostAllocateFFP';
-import { updateActivity } from '../../actions/activities';
+import {
+  setCostAllocationFFPFundingSplit,
+  setCostAllocationFFPOtherFunding
+} from '../../actions/editActivity';
 
-describe('the CostAllocateFFP component', () => {
-  const sandbox = sinon.createSandbox();
+jest.spyOn(React, 'useContext');
+React.useContext.mockImplementation(() => ({
+  index: 'activity index'
+}));
```

Comment:
I am not entirely sure I need this; I copied it from the Contractor Resources tests in an attempt to get things to work.

Reply:
Nah, you don't need this anymore!